SharedSources/MediaLibraryService.swift

```diff
@@ -295,6 +295,10 @@ extension MediaLibraryService {
         medialib.reload()
     }
 
+    @objc func reindexAllMediaForSpotlight() {
+        media(ofType: .video).forEach { $0.updateCoreSpotlightEntry() }
+        media(ofType: .audio).forEach { $0.updateCoreSpotlightEntry() }
```

Comment:
I could also have that code directly in the settingscontroller but having this method makes it clearer what is happening